src/YAMLValidator/InputProcessor.cs

```diff
@@ -54,7 +54,7 @@ public static RootCommand GetRootCommand()
         var schemaOption = new Option<string>(
             name: "--schema",
             description: "The path to the schema json file",
-            getDefaultValue: () => @"..\schemas\pa-yaml\v3.0\pa.yaml-schema.json"
+            getDefaultValue: () => @".\schema\pa.yaml-schema.json"
```

Comment:
nit: I'd keep the version number

Reply:
just to clarify, I put that as a dummy filepath since this was just the scaffolding. The actual .json schema files are from the [docs folder from the root of the repo](https://github.com/microsoft/PowerApps-Tooling/tree/master/docs)

Reply:
I know, my was just a suggestion.